src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -62,18 +63,18 @@ class IntroductionPage extends React.Component {
               <div><h6>When you apply for a disability increase, be sure to have these on hand:</h6></div>
               <ul>
                 <li>Your Social Security number</li>
-                <li>VA medical and hospital records that show your claimed disability has gotten worse</li>
-                <li>Private medical and hospital records that show your claimed disability has gotten worse</li>
+                <li>VA medical and hospital records that show your rated disability has gotten worse</li>
+                <li>Private medical and hospital records that show your rated disability has gotten worse</li>
               </ul>
-              <p>In some situations you may need to turn in additional forms to support your claim. For example, you’ll need to fill out another form if you’re claiming a dependent or requesting aid and attendance benefits. <a href="#">Learn more about what additional forms you may need to file with your disability claim</a>.</p>
+              <p>In some cases, you may need to turn in one or more supplemental forms to support your claim. For example, you’ll need to fill out another form if you’re claiming a dependent or applying for aid and attendance benefits.<br/> <a href="https://github.com/department-of-veterans-affairs/vets-website/pull/disability-benefits/apply/supplemental-forms/">Learn what additional forms you may need to file with your disability claim</a>.</p>
```

Comment:
Is the non-functional link intentional here? If so, I'd vote for leaving a blank link to make it more obvious or adding a TODO

Reply:
Great question @markgreenburg, @peggygannon do we have a link we can provide here?

Reply:
@cehsu @markgreenburg Yes, the link should be: 
/disability-benefits/apply/supplemental-forms/

this page/link is not live yet. This page will launch when we launch the 526 MVP.